plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1409,9 +1410,11 @@ public Optional<ConnectorOutputMetadata> finishCreateTable(
             if (handle.readVersion().isPresent()) {
```

Comment:
Change commit title to fit within 80 characters

`Write checkpoint if necessary for CREATE OR REPLACE statements`

Reply:
Done ✅